File: Selenium/Project/Activity4.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
    driver.get("https://alchemy.hguy.co/crm")
    userName = driver.find_element(By.ID,"user_name")
    userName.send_keys("admin")
    
    password = driver.find_element(By.ID,"username_password")
    password.send_keys("pa$$w0rd")
	
    login = driver.find_element(By.ID,"bigbutton")
    login.click()
    
    dashBoard = driver.find_element(By.ID,"tab0")
    
    print(f"Dashboard text is {dashBoard.text} ")
    assert dashBoard.is_displayed(),"Homepage is not displayed , login not successfull"
    print("Login is successsfull and homepage is opened")
    driver.quit()
    

File: Selenium/Project/Activity4.java
package seleniumActivities;

import static org.testng.Assert.assertEquals;
import static org.testng.Assert.assertTrue;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity4 {
	WebDriver driver;
	
	@BeforeClass
	public void setUp() {
		driver = new FirefoxDriver();
		driver.get("https://alchemy.hguy.co/crm");
	}
	
	@Test(priority = 0)
	public void login() {
		WebElement userName = driver.findElement(By.id("user_name"));
		WebElement password = driver.findElement(By.id("username_password"));
		WebElement login = driver.findElement(By.id("bigbutton"));
		
		userName.sendKeys("admin");
		password.sendKeys("pa$$w0rd");
		login.click();
	}
	
	@Test(priority = 1 , dependsOnMethods = "login")
	public void homePage() {
		WebElement dashBoard = driver.findElement(By.id("tab0"));
		System.out.println("Dashboard text is: "+dashBoard.getText());
		assertTrue(dashBoard.isDisplayed(),"Homepage is not displayed , login not successfull");
		System.out.println("Login is successsfull and homepage is opened");
	}
	
	@AfterClass
	public void tearDown() {
		driver.quit();
	}

}
